### PR TITLE
AB#80623 remove sites legacy code

### PR DESCRIPTION
**Pull request by jakub-lotko**

### CODE REVIEW

File: src/HE.InvestmentLoans.BusinessLogic/LoanApplication/CommandHandlers/StartApplicationCommandHandler.cs [hunk 1]
```diff
@@ -42,8 +47,11 @@ public class StartApplicationCommandHandler : IRequestHandler<StartApplicationCo
                     new[] { new ErrorItem(nameof(LoanApplicationName), "This name has been used for another application") },
                     null);
             }
+        await _applicationRepository.Save(newLoanApplication, await _loanUserContext.GetUserDetails(), cancellationToken);
```
Comment (jjankowski87) [inline src/HE.InvestmentLoans.BusinessLogic/LoanApplication/CommandHandlers/StartApplicationCommandHandler.cs:50]: `newLoanApplication` is saved twice, is that correct?
also looks like formatting is broken after merge

File: src/HE.InvestmentLoans.BusinessLogic/LoanApplication/Repositories/LoanApplicationRepository.cs [hunk 2]
```diff
@@ -63,6 +66,26 @@ public async Task<bool> IsExist(LoanApplicationName loanApplicationName, UserAcc
         return response.invln_loanexists;
     }
 
+    public async Task<LoanApplicationViewModel> GetLoanApplicationDetails(LoanApplicationId id, UserAccount userAccount, CancellationToken cancellationToken)
```
Comment (jjankowski87) [inline src/HE.InvestmentLoans.BusinessLogic/LoanApplication/Repositories/LoanApplicationRepository.cs:69]: Should repository return View Model?

Comment (Carq) [reply src/HE.InvestmentLoans.BusinessLogic/LoanApplication/Repositories/LoanApplicationRepository.cs:69]: Nope

File: src/HE.Investments.IntegrationTests/Loans/Application/WithdrawDraftApplicationIntegrationTests.cs [hunk 3]
```diff
@@ -23,7 +23,7 @@ public WithdrawDraftApplicationIntegrationTests(IntegrationTestFixture<Program>
         _applicationLoanId = UserData.LoanApplicationIdInDraftState;
     }
 
-    [Fact(Skip = LoansConfig.SkipTest)]
+    [Fact(Skip = "LoansConfig.SkipTest")]
```
Comment (Carq) [inline src/HE.Investments.IntegrationTests/Loans/Application/WithdrawDraftApplicationIntegrationTests.cs:26]: Revert?

**Review by Carq — APPROVED** [review]

**Review by dawid-pawulski — APPROVED** [review]

**Pull request merged**